content/hardware/02.hero/boards/uno-r4-minima/tutorials/usb-hid/usb-hid.md

```diff
@@ -15,10 +15,10 @@ This feature can be used to create game controllers, keyboard extensions or othe
 
 ## Goals
 
-The goals of this tutorials are:
+The goals of this tutorial is:
```

Comment:
This should be "are" since it is multiple learning goals

Reply:
we are mixing yes let me fix it

Reply:
you are right